x-pack/plugins/siem/server/graphql/ecs/schema.gql.ts

```diff
@@ -10,39 +10,39 @@ export const ecsSchema = gql`
   scalar ToStringArray
 
   type EventEcsFields {
-    action: String
-    category: String
-    created: Date
```

Comment:
Should this field stay as the gql `Date` scalar? And if not, should it be an array of the `Date` scalar?

Reply:
Same question pertains to `end: Date` and `start: Date` below as well.

Reply:
Since that is one of the GraphQL errors keeping us from moving forward and other people are parsing out strings, I would suggest we make it into a `ToStringArray` and then re-visit the Date Scalar to see if we can get it return arrays of dates as well. Up to you on that one.

I don't think we use that at the moment for anything other timeline renderer so it should only help things at this point unless you see something I don't. Then please let me know.

Reply:
Ok so we now have `DateToArray` and all these should be fixed!